core/src/main/java/org/apache/gravitino/StringIdentifier.java

```diff
@@ -219,9 +219,9 @@ public static String removeIdFromComment(String comment) {
       }
 
       if (indexOf != -1) {
-        return comment.substring(0, indexOf);
+        return comment.substring(0, indexOf).trim();
       }
     }
-    return comment;
+    return comment.trim();
```

Comment:
This would change the comment even if it didn't have an identifier in it, I'm not sure we want to do that.